packages/Switch/src/Switch.styles.js

```diff
@@ -0,0 +1,158 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const KnobPositionStyles = {
+  small: `
+    transform: translateX(12px);
+  `,
+  medium: `
+    transform: translate(20px, 0);
+  `,
+  large: `
+    transform: translate(28px);
+  `,
+};
+
+const KnobSizeStyles = {
+  small: `
+    height: 12px;
+    width: 12px;
+
+    &::after {
+      height: 13px;
+    }
+  `,
+  medium: `
+    height: 20px;
+    width: 20px;
+
+    &::after {
+      height: 21px;
+    }
+  `,
+  large: `
+    height: 28px;
+    width: 28px;
+
+    &::after {
+      height: 29px;
+    }
+  `,
+};
+
+const UnderlaySizeStyles = {
+  small: `
+    height: ${stylers.spacer(2)};
+    width: 28px;
+  `,
+  medium: `
+    height:  ${stylers.spacer(3)};
+    width: 44px;
+  `,
+  large: `
+    height:${stylers.spacer(4)};
+    width: 60px;
+  `,
+};
+
+export const KnobStyled = styled.span`
```

Comment:
should we just export the raw styles here and apply them to a span defined in the component itself? Just curious because I'm not sure if we standardize this

Reply:
Good question, I'm not sure as well 😄 
From my understanding, if we want to read `props` is easier to use `css` prop. If it's just plain css, it's cleaner to just import `ElementStyled`

Reply:
Yeah, the approach we've been trying out has been to export the raw styles and use the `css` prop to apply them to an element / component. Maybe @allison-c had a reason for this approach though?